keyvi/src/cpp/dictionary/dictionary_merger.h

```diff
@@ -100,9 +100,29 @@ final {
     };
 
 public:
-    DictionaryMerger(size_t memory_limit = 1073741824, const merger_param_t& params = merger_param_t())
```

Comment:
@hendrikmuhs 
What you think, if we change memory limit units from `byte` for `MB` ?